install.py

```diff
@@ -145,6 +145,14 @@ def go(ui, args, answerfile_address, answerfile_script):
         elif opt == "--netinstall":
             results['netinstall'] = True
             logger.log("This is a netinstall.")
+        # XCP-ng: no-repo-gpgcheck
+        elif opt == "--no-repo-gpgcheck":
```

Comment:
The boot parameters (and possibly the options?) should be documented in `doc/parameters.txt`

Reply:
Added in new revision

Reply:
I mentioned the boot parameters in priority, rather that the options handled by init.py, as that's the most useful for users. But I'm not sure what the logic is to decide what options gets described in the boot parameters and what options are described in the script parameters.

Reply:
It looks like there is all installer-specific option are described under "Installer", though the "Startup" section is a mix of options interpreted by kernel and scripts launched directly by `init` (I mean `systemd` here, not the installer's own `init` script :dizzy_face:)